src/app/services/http/http.service.ts

```diff
@@ -0,0 +1,22 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HttpService {
+  constructor(private http: HttpClient) {
+    this.serverUrl = "https://my-json-server.typicode.com/miladsoft/chat-db/";
```

Comment:
What's this, some test DB? Should be removed, this whole http folder and service.